packages/BUILD.bazel

```diff
@@ -410,6 +410,17 @@ latex_package(
     tests = ["glossaries-extra_test.tex"],
 )
 
+latex_package(
+    name = "gnuplottex",
+    srcs = [
+        "@texlive_texmf__texmf-dist__tex__latex__gnuplottex",
+        "@texlive_texmf__texmf-dist__tex__latex__moreverb",
+        ":verbatim",
+        ":graphicx",
```

Comment:
Switch the internal and external packages around, to adhere to the style used in this BUILD.bazel file.
```
        ":graphicx",
        ":verbatim",
        "@texlive_texmf__texmf-dist__tex__latex__gnuplottex",
        "@texlive_texmf__texmf-dist__tex__latex__moreverb",
```

Reply:
Done, thanks!